### DISCUSSION TITLE
New R projects don't create an .Rproj file

### DISCUSSION THREAD

**Discussion by clarsenEmory** [posit-dev/positron#3967] [Q&A]
New R projects don't seem to create an .Rproj file which happens in Rstudio.  Is there a place to make this happen as default behavior? 
Also is there support for quarto projects that is similar to Rstudio?

**Comment by sharon-wang** [comment]
Hi Christian! Thanks for raising these questions :)

### Q: "New R projects don't seem to create an .Rproj file which happens in Rstudio. Is there a place to make this happen as default behavior?"

I think the workspace [`settings.json`](https://code.visualstudio.com/docs/getstarted/settings#_workspace-settings) in Positron/VSCode land is an analog to the `.Rproj` created by RStudio, in that it can help store some settings/configuration specific to a project.

As long as you have a folder open in Positron, you can configure the workspace settings. One way to get to the settings is by clicking the Gear icon in the bottom-left corner.

<img width="318" alt="positron_settings" src="https://github.com/posit-dev/positron/assets/25834218/7b515226-c2d1-4c2a-8f8f-03e5598274cc">

From there, you can click on "Workspace" and search for settings and modify them. For example, I searched "positron r language pack" to look at R-specific settings.

<img width="896" alt="positron_settings_r_lang_pack" src="https://github.com/posit-dev/positron/assets/25834218/89530034-51d4-475a-9f13-d68f1c27491e">

Are there particular settings you use in an `.Rproj` file that you don't see an equivalent for in `settings.json`?

### Q: "Also is there support for quarto projects that is similar to Rstudio?"

In terms of the Project Wizard, we are tracking interest for other project types here: https://github.com/posit-dev/positron/issues/3364, including a few different Quarto project types. Please feel free to chime in with any specific functionality you'd like to see in this discussion or the linked issue!
